services/Meetup.json

```diff
@@ -3,17 +3,11 @@
     "documents": {
         "tos": {
             "location": "https://help.meetup.com/hc/en-us/articles/360027447252",
-            "contentSelector": ".article",
-            "filters": [
-                "removeFooter"
```

Comment:
This filter was necessary to remove the link at the end of the document 
`[Follow](/hc/en-us/articles/360027447252-Terms-of-Service/subscription.html "Opens a sign-in dialog")`